accessibility/core/src/main/java/com/novoda/accessibility/Actions.java

```diff
@@ -0,0 +1,52 @@
+package com.novoda.accessibility;
+
+import android.support.annotation.IdRes;
+import android.support.annotation.Nullable;
+
+import java.util.List;
+
+public class Actions {
```

Comment:
If this implemented `Iterable<Action>`, you could use a foreach in https://github.com/novoda/spikes/pull/20/files#diff-4f813c698d6722c775ecff012bc627a0R22